### PR TITLE
Add 3 default MISP feeds: CIRCL, ThreatFox, Botvrij

### PR DESCRIPTION
**Pull request by ckane**
### Proposed changes

***Update***: Added 3 sample `config.yml` files. The platform admin can decide how they'd like to deploy the local copies of `misp-feed` to consume whatever feeds that they want to consume.

<strike>Modify `external-import/misp-feed/src/misp-feed.py` so that it will read the `config.yml` from the current working directory, rather than trying to find the one that is located in the same folder as the `misp-feed.py` script.

Add three subdirectories to the misp-feed `src/` folder that contain config templates for the following MISP-format feeds:
* https://www.circl.lu/doc/misp/feed-osint
* https://www.botvrij.eu/data/feed-osint
* https://threatfox.abuse.ch/downloads/misp/

A proposal for the use would be to (more or less) run each like this:
```sh
cd connectors/external-import/misp-feed/src/circl
python3 ../misp-feed.py
```
```sh
cd connectors/external-import/misp-feed/src/botvrij
python3 ../misp-feed.py
```
```sh
cd connectors/external-import/misp-feed/src/threatfox
python3 ../misp-feed.py
```

In my environment, I have written 3 `systemd` services that execute each feed derivative like above from within each of the working directories, but likely some further modification would need to be made to the `docker-compose` code to run all three of these in a single docker, or multiple docker, instances (depending upon what the preference is for the team) to support the containerized approach.

The hope here is that providing these as part of the codebase will make it a lot easier & quicker for new users to install a system that starts populating itself with well-known OSINT datasets out there.

Another key piece here was that I wanted there to remain a single `misp-feed.py` script that all MISP feeds would use, rather than having to make copies of it into new connector folders for each feed, and then re-copy it every time fixes are released from the project.</strike>

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality using different use cases

### CODE REVIEW

File: external-import/misp-feed/src/misp-feed.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ class Text:
 class MispFeed:
     def __init__(self):
         # Instantiate the connector helper from config
-        config_file_path = os.path.dirname(os.path.abspath(__file__)) + "/config.yml"
```
Comment (SamuelHassine) [inline external-import/misp-feed/src/misp-feed.py]: Hey @ckane why did you remove this mecanism which is present in all connectors?

**Review by SamuelHassine — COMMENTED** [review]
Can you please remove the folders and just create additional configuration files like: `config.botvrij.yml.sample`?

Thanks!

File: external-import/misp-feed/src/misp-feed.py (hunk 1 above)
Comment (ckane) [reply external-import/misp-feed/src/misp-feed.py]: The choice to remove this was so that the single copy of `misp-feed.py` could remain in the present folder. The current mechanism forces the script to use the `config.yml` that is co-located with it in the `misp-feed/` folder, which means I cannot run 3 instances of the same script with different `config.yml` files located in different folders and all 3 instance will only look at the `config.yml` located in the same directory as the Python script.

Making the above change allows me to `cd` into a different working directory, before calling each instance of the `misp-feed.py` script, and therefore have 3 running copies of the same module that pull down 3 feeds and run as 3 different connectors.

**Comment by ckane** [comment]
> Can you please remove the folders and just create additional configuration files like: `config.botvrij.yml.sample`?
> 
> Thanks!

Yes, I can do this if you'd prefer it to be that way.

File: external-import/misp-feed/src/misp-feed.py (hunk 1 above)
Comment (ckane) [reply external-import/misp-feed/src/misp-feed.py]: I'll revert this change too, but I think it would be worth discussing whether all the connectors should behave this way, or if it should just look for the `config.yml` from the current working directory, and leave it up to the platform administrator to decide if they want the current working directory to be the same as the script's containing folder.

**Comment by ckane** [comment]
Ok @SamuelHassine - I've re-done the change according to your suggestion, let me know if this works for you

**Review by SamuelHassine — APPROVED** [review]
Thanks a lot!
